Duplicati/GUI/Duplicati.GUI.TrayIcon/Program.cs

```diff
@@ -318,8 +318,8 @@ private static TrayIconBase RunTrayIcon(string toolkit)
         // attempt to load non-existing assemblies
 
         [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
-        private static TrayIconBase GetWinformsInstance() { return new Windows.WinFormsRunner(); }
-#if __WindowsGTK__ || ENABLE_GTK
+        private static TrayIconBase GetWinformsInstance() { return new Win32Runner(); }
+#if __MonoCS__ || __WindowsGTK__ || ENABLE_GTK

```

Comment:
These references (`__MonoCS__`) were removed in another recent PR - since this particular PR has been outstanding for so long, I wonder if there are any other changes that need to be incorporated back from master...

Reply:
TrayIcon (Windows version) was rewritten to pure Win32, so there could be some small leftovers.